nix/stack-shell.nix

```diff
@@ -3,7 +3,7 @@
 { system ? builtins.currentSystem
```

Comment:
When I try `stack --nix build`, it starts to build GHC 8.4.3 (`/nix/store/7sih6g9zyzbrjc0qqgq5692m3pwgazkz-ghc-8.4.3.drv`). Why is that?